administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -0,0 +1,241 @@
+
+	<div id="j-main-container">
+
+	<div class="js-stools clearfix">
+		<div class="clearfix">
+			<div class="js-stools-container-bar">
+
+				<div class="js-stools-field-filter js-stools-menutype">
+					<select id="component" name="component">
+						<option value="">- Select Item type -</option>
+						<optgroup label="Content">
+							<option value="com_content.articles">- Articles</option>
+							<option value="com_content.categories">- Categories</option>
+						</optgroup>
+						<optgroup label="Contacts">
+							<option value="com_contact.contacts">- Contacts</option>
+							<option value="com_contact.categories">- Categories</option>
+						</optgroup>
+					</select>
+				</div>
+
+				<div class="js-stools-field-filter js-stools-menutype">
+					<select id="ref-language" name="ref-language">
+						<option value="">- Select Reference Language -</option>
+						<?php echo JHtml::_('select.options', JHtml::_('contentlanguage.existing', false, true), 'value', 'text'); ?>
+					</select>
+				</div>
+			<?php if ($assoc && $input->get('layout') != 'modal') : ?>
+				<button id="filter-submit" class="btn btn-primary" type="submit" title="<?php echo JText::_('MOD_MULTILANGSTATUS'); ?>">
+					<span class="icon-list "> </span> 
+					Select
+				</button>
+			<?php endif; ?>
+		</div>
+	</div>
+
+	<?php if (!empty($results)) : ?>
+		<table class="table table-striped" id="contactList">
+			<thead>
+				<tr>
+					<th class="nowrap">
+						<?php echo JHtml::_('searchtools.sort', 'JGLOBAL_TITLE', 'a.name', $listDirn, $listOrder); ?>
+					</th>
+					<th width="15%" class="nowrap hidden-phone">
+						<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_LANGUAGE', 'language_title', $listDirn, $listOrder); ?>
+					</th>
+					<?php if ($assoc) : ?>
+					<th width="5%" class="nowrap hidden-phone hidden-tablet">
+						<?php echo JHtml::_('searchtools.sort', 'COM_ASSOCIATIONS_HEADING_ASSOCIATION', 'association', $listDirn, $listOrder); ?>
+					</th>
+					<?php endif;?>
+					<th width="1%" class="nowrap hidden-phone">
+						<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ID', 'a.id', $listDirn, $listOrder); ?>
+					</th>
+				</tr>
+			</thead>
+			<tfoot>
+				<tr>
+					<td colspan="10">
+						<?php //echo $this->pagination->getListFooter(); ?>
+					</td>
+				</tr>
+			</tfoot>
+			<tbody>
+			<?php
+			$n = count($this->items);
+			foreach ($results as $i => $item) :
+				$canCreate  = $user->authorise('core.create',     'com_menus.menu.' . $menutypeid);
+				$canEdit    = $user->authorise('core.edit',       'com_menus.menu.' . $menutypeid);
+				$canCheckin = $user->authorise('core.manage',     'com_checkin') || $item->checked_out == $user->get('id')|| $item->checked_out == 0;
+				$canChange  = $user->authorise('core.edit.state', 'com_menus.menu.' . $menutypeid) && $canCheckin;
+				?>
+				<tr class="row<?php echo $i % 2; ?>" sortable-group-id="<?php echo $item->catid; ?>">
```

Comment:
is this view sortable on drag by categopry id? no, so remove the `sortable-group-id` html attribute.